SPECS/python-gevent/python-gevent.spec

```diff
@@ -64,6 +59,8 @@ nosetests
 %{python3_sitelib}/*
 
 %changelog
+* Mon Nov 27 2023 Tobias Brick <tobiasb@microsoft.com> - 23.9.1-1
+
```

Comment:
Please add 1 line in the changelog entry describing this is an update to 23.9.1

Reply:
I really thought I had. Weird; thanks.